tests/test_in/config_example.json

```diff
@@ -2,19 +2,19 @@
     "sphinx": {
         "enabled": true,
         "cq_default_path": "doc/source/conf.py",
-        "min": 0,
-        "max": 0
+        "min": "$MIN_SPHINX_WARNINGS",
+        "max": "${MAX_SPHINX_WARNINGS}"
```

Comment:
brackets or not? Maybe the flow we could use something standard from YML syntax?

Reply:
Curly braces are optional, which is what I tried to demonstrate here. It should be a [`string.Template`](https://docs.python.org/3/library/string.html#template-strings).

Reply:
OK, makes sense. Lets share this info in the README as well - since that is what people will mostly read.